### PR TITLE
add no var rule to eslint

### PR DESCRIPTION
**Pull request by DarylThayil**
and update instances of it

### CODE REVIEW

**Comment by pkanher617** [comment]
Seems like there are some duplicates with #934?

**Comment by DarylThayil** [comment]
@pkanher617  this enforces a different rule, it says no var, and the fix is turning them from var to either let or const

File: lib/msal-core/.eslintrc.json [hunk 1]
```diff
@@ -26,8 +26,8 @@
         "@typescript-eslint/no-object-literal-type-assertion": 0,
         "@typescript-eslint/prefer-interface": 0,
         "@typescript-eslint/type-annotation-spacing": 0,
-        "no-var": 0,
-        "prefer-const": 0,
+        "no-var": 2,
+        "prefer-const": 2,
```
Comment (jasonnutter) [inline lib/msal-core/.eslintrc.json]: Looks like this is doing both rules, I'm guessing you created this branch from the other branch?

Comment (jasonnutter) [reply lib/msal-core/.eslintrc.json]: I think pulling master into this branch will clean this up.

Comment (DarylThayil) [reply lib/msal-core/.eslintrc.json]: oh snap, updated

**Pull request merged**